kbase-extension/static/kbase/js/widgets/appWidgets2/view/sequenceView.js

```diff
@@ -151,39 +127,19 @@ define([
                     },
                 });
 
-                const postButton = div(
```

Comment:
This button just sits there, invisible, unless you mouse over it. Then it becomes a pointer and behaves like a button... but it's not actually tied to anything.

So, just remove it and tweak the bootstrap classes so it's no longer an input group.